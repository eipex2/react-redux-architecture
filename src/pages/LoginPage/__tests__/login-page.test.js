import React from 'react'
import { shallow } from 'enzyme'

import LoginView from '../login-view'

describe('[<LoginPage />]', () => {

    it('shd render <LoginView /> and view shd match snapshot', () => {
        const wrapper = shallow(<LoginView />)

        expect(wrapper.getElement()).toMatchSnapshot()
    })
})