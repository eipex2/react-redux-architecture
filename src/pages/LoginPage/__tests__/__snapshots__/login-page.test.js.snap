// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[<LoginPage />] shd render <LoginForm /> 1`] = `
<Page
  className="login_view"
>
  <LoginForm />
</Page>
`;

exports[`[<LoginPage />] shd render <LoginView /> 1`] = `
<Page
  className="login_view"
>
  <LoginForm />
</Page>
`;

exports[`[<LoginPage />] shd render <LoginView /> and view shd match snapshot 1`] = `
<Page
  className="login_view"
>
  <LoginForm />
</Page>
`;
